features/networking/operator.feature

```diff
@@ -345,3 +345,25 @@ Feature: Operator related networking scenarios
     And all existing pods die with labels:
       | app=dhcp-daemon |
     """
+
+  # @author anusaxen@redhat.com
+  # @case_id OCP-27333
+  @admin
+  @destructive
+  Scenario: Changing mtu in CNO should not be allowed
+  Given the mtu value "1750" is patched in CNO config according to the networkType
```

Comment:
We use one more level indention for the steps.

Reply:
Do we need to restore the change after scenario ?

Reply:
Restore is being taken care in same step via ```teardown_add``` procedure